### PR TITLE
Auto-Remediation scripts changes

### PR DESCRIPTION
**Pull request by saurabhgautam-msft**

### CODE REVIEW

File: Scripts/RemediationScripts/ControlRemediationSpecification.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "Description": "Specifies metadata about Remediation scripts",
+    "RemediationScriptsBaseUrl": "https://aka.ms/azts/remediation/",
+    "RemediationSpecification": [
+        {
+            "ControlId": "Azure_Storage_AuthN_Dont_Allow_Anonymous",
+            "EnableRemediation": "true",
+            "RemediationScriptUrl": "https://raw.githubusercontent.com/azsk/AzTS-docs/users/sagautam/feature/auto_remediation/Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1",
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/ControlRemediationSpecification.json:8]: Change to main branch

File: Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1 [hunk 2]
```diff
@@ -212,25 +212,33 @@ function Remove-AnonymousAccessOnContainers
...
         [Parameter(ParameterSetName = "RemediationAtStorageLevel", HelpMessage="Specifies the path to the file to be used as input for the remediation")]
-        $FilePath
+        $FilePath,
+
+        [switch]
+        [Parameter(Mandatory = $false)]
+        $AutoRemediation,
+
+        [string]
+        [Parameter(Mandatory = $false)]
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1:222]: please make 'T' uppercase and add help text for both new params.

File: Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1 [hunk 3]
```diff
@@ -467,61 +560,96 @@ function Remove-AnonymousAccessOnContainers
...
+                                Write-Host $([Constants]::SingleDashLine)
+                                # Write this to a file.
+                                $storageAccountsSkippedFile = "$($folderpath)\SkippedStorageAccountsFromRemediation.csv"
+                                $skippedStorageAccountsFromRemediation | Export-CSV -Path $storageAccountsSkippedFile -NoTypeInformation
+                                Write-Host "The information of Storage accounts where 'Allow Blob Public Access' has not been successfully disabled is saved to $($storageAccountsSkippedFile)"
+                                
+                            }
                         }
-                        
-                        return
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1:501]: Why this return statement removed this was in normal flow path?

File: Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1 [hunk 4]
```diff
@@ -693,12 +823,12 @@ function Set-AnonymousAccessOnContainers
     )
 
     Write-Host $([Constants]::DoubleDashLine)
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1:825]: Noramlly ... three dots represent something in progress, so I will suggest lets keep them as any how rollback will not be run from UI generated script. Thoughts?

File: Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1 (hunk 3 above)
Comment (saurabhgautam-msft) [reply Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1:501]: It was not necessary, so removed it.

File: Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1 (hunk 2 above)
Comment (saurabhgautam-msft) [reply Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1:222]: Done

File: Scripts/RemediationScripts/ControlRemediationSpecification.json (hunk 1 above)
Comment (saurabhgautam-msft) [reply Scripts/RemediationScripts/ControlRemediationSpecification.json]: Done

**Review by TarunKrShukla — CHANGES_REQUESTED** [review]
Update Instructions.md file

**Review by TarunKrShukla — APPROVED** [review]

**Pull request merged**